errors/no-html-link-for-pages.md

```diff
@@ -40,6 +40,24 @@ function Home() {
 export default Home
 ```
 
+### Options
+
+#### `pagesDir`
+
+This rule can normally locate your `pages` directory automatically.
+
+If you're working in a monorepo, we recommend configuring the [`rootDir`](https://nextjs.org/docs/basic-features/eslint.md#rootDir) setting in `eslint-plugin-next`, which this rule will use to locate your `pages` directory.
```

Comment:
```suggestion
If you're working in a monorepo, we recommend configuring the [`rootDir`](/docs/basic-features/eslint.md#rootDir) setting in `eslint-plugin-next`, which `pagesDir` will use to locate your `pages` directory.
```

Reply:
Does this first section need an example, if the second one does?

Reply:
We could add it here, but I worry that users would miss the details and configuration options - WDYT?